tools/toolchain/clang_lto_pgo.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -uex
+
+if [ -d "/optimized_clang" ];
+then
+	DIR="/optimized_clang"
+else
+	DIR="${PWD}/optimized_clang"
+	mkdir -p "${DIR}"
+fi
+cd "${DIR}"
+
+# Download the scylla codebase for training.
+# We assume that Scylla's dependencies are already installed.
+git clone https://github.com/scylladb/scylla --depth=1
+git -C scylla submodule update --init --depth=1
+
+# Clone, patch and bootstrap the newest Clang and BOLT.
+git clone https://github.com/llvm/llvm-project --branch main --depth=1 stage-0
```

Comment:
I think it's better to stick to a single version, to avoid things compiling for a developer, then not compiling during integration.

Reply:
We can certainly stick to a released version and update it once in a while.